ui/i18n/datepicker-ko.js

```diff
@@ -25,7 +25,7 @@ datepicker.regional.ko = {
 	dayNamesShort: [ "일","월","화","수","목","금","토" ],
 	dayNamesMin: [ "일","월","화","수","목","금","토" ],
 	weekHeader: "주",
-	dateFormat: "yy-mm-dd",
+	dateFormat: "yy. mm. dd.",
```

Comment:
Please note (from the wiki): "Compared to master, the default date format is different, is that intended? Master has 4 digit years, rewrite only 2"